package/doc/sphinx/source/documentation_pages/selections.rst

```diff
@@ -300,14 +300,29 @@ bynum *index-range*
     selects atoms 5 through 10 inclusive. All atoms in the
     :class:`MDAnalysis.Universe` are consecutively numbered, and the index
     runs from 1 up to the total number of atoms.
-
+    
+id *index-range*
+    selects all atoms in the given range which consists of two numbers 
+    separated by a colon such as ``id 5:10`` (both inclusive),
+    e.g. ``id 1`` selects all the atoms with id 1; ``id 5:7`` selects all
+    atoms with ids 5, all atoms with ids 6 and all atoms with ids 7.
+     
 index *index-range*
     selects all atoms within a range of (0-based) inclusive indices,
     e.g. ``index 0`` selects the first atom in the universe; ``index 5:10``
     selects atoms 6 through 11 inclusive. All atoms in the
     :class:`MDAnalysis.Universe` are consecutively numbered, and the index
     runs from 0 up to the total number of atoms - 1.
-
+    
+    
+.. note::
```

Comment:
I think I would rephrase the note similar to this:
Conventionally, ``id`` corresponds to the serial number in PDB format. In contrast to ``bynum``, the ``id`` index is not necessarily continuous nor unique, and can be arbitrarily assigned by the user. Therefore, different atoms in the :class:`MDAnalysis.Universe` can share the same ``id`` and the user can set the ``id`` index to an arbitrary number.
What do you think? Do you think there is somethin missing here?